lib/mongodb.js

```diff
@@ -797,6 +797,33 @@ function hasNearFilter(where) {
   return false;
 }
 
+MongoDB.prototype.buildLookup = function (relation, include, model) {
+  var self = this;
```

Comment:
Yes, that is correct. the current change creates a  `$lookup`  - that is left join for mongodb - for each relation that is included.
for more details refer to mongodb documentation
https://docs.mongodb.com/v3.2/reference/operator/aggregation/lookup/